prepare_royvanrijn.sh

```diff
@@ -16,7 +16,7 @@
 #
 
 source "$HOME/.sdkman/bin/sdkman-init.sh"
-sdk use java 21.0.1-graal 1>&2
+sdk use java 21.0.2-graal 1>&2
```

Comment:
Wow, so cool to see how @hundredwatt's work is paying of: running _./evaluate.sh royvanrijn_ automatically installs that new version now, without anything to be done by hand. Noice!